xrfeitoria/renderer/renderer_unreal.py

```diff
@@ -155,6 +155,9 @@ def render_jobs(cls) -> None:
         # post process, including: convert cam params.
         cls._post_process()
 
+        # clear render queue
+        cls.clear()
+
```

Comment:
- add this clear info in the docstring of this function (also applicable for renderer_blender)
- delete line 152

Reply:
fixed in e536fc2